calc/calc.py

```diff
@@ -86,27 +158,58 @@ def _next_token(self):
         # Get the character that's at the current position.
         current_character = text[position]
 
+        # If we find whitespace eat it like we're Packman.
+        if current_character.isspace():
+            self._consume_whitespace(text)
+
+            if self.position > len(text) - 1:
+                return Token(EOF, None)
+            # After we've eatten all the bab whitespace lets actually grab a
+            # symantically meaningful character.
+            current_character = text[self.position]
+
+        # Saved By The Test: test_arithmetic caught an error if return_token
+        #                    was never set
+        return_token = None # Stores the token that's been lexed.
```

Comment:
Why call this `return_token`? Why not just `token`? We didn't `import token` above, so that's not being used as a variable name.